crt_portal/cts_forms/urls.py

```diff
@@ -6,8 +6,7 @@
 app_name = 'crt_forms'
 
 urlpatterns = [
-    path('', views.IndexView.as_view(), name='index'),
-    path('<int:pk>/', views.DetailView.as_view(), name='detail'),
-    path('<int:pk>/results/', views.ResultsView.as_view(), name='results'),
-    path('<int:violationreport_id>/vote/', views.VoteView, name='vote'),
+    # path('', views.IndexView.as_view(), name='index'),
```

Comment:
@LindsayYoung No URLs for `cts_forms`?

Reply:
We haven't made them yet, the ones we took away were from the prototype.

Reply:
@LindsayYoung I see! Without any code comments the intention here isn't clear. I think a code comment about the intention would be helpful, but not necessarily a blocker.